spec/checkPullRequestLabelsSpec.js

```diff
@@ -199,7 +230,7 @@ describe('Pull Request Label Check', () => {
         body:
           'Hi, @' +
           payloadData.payload.pull_request.user.login +
-          '. This pull request does not have a "CHANGELOG: ..." label ' +
+          ', This pull request does not have a "CHANGELOG: ..." label ' +
```

Comment:
After the comma, the `This` should not start with capital letters.

Reply:
Updated.